helm-charts/falcon-kac/templates/deployment_webhook.yaml

```diff
@@ -92,6 +92,23 @@ spec:
         {{- end }}
       {{- end }}
     spec:
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: node-role.kubernetes.io/control-plane
+                operator: DoesNotExist
+              - key: node-role.kubernetes.io/master
+                operator: DoesNotExist
+              - key: kubernetes.io/os
+                operator: In
+                values:
+                - linux
```

Comment:
This duplicates `nodeSelector`
https://github.com/CrowdStrike/falcon-helm/blob/main/helm-charts/falcon-kac/templates/deployment_webhook.yaml#L209-L210

Reply:
Dropped. See previous comments.